### PR TITLE
Extract `TimeRange` class

### PR DESCRIPTION
**Pull request by seanpdoyle**
Encapsulates the logic around time-range formatting.

### CODE REVIEW

**Comment by desi** [comment]
Nice refactor. I like it! :+1:

File: app/models/time_range.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @end_at = end_at
+  end
+
+  def to_s
+    I18n.t(
+      "time_ranges.format",
+      weekday: weekday.label,
+      start_at: format(start_at),
+      end_at: format(end_at),
+    ).squish
```
Comment (gabebw) [inline app/models/time_range.rb]: Why `squish`? What does it look like before squishing, and what does it look like after?

File: app/models/time_range.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+class TimeRange
```
Comment (gabebw) [inline app/models/time_range.rb]: Add tests for this class?

File: app/models/time_range.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  FORMAT = "%l:%M %P".freeze
+
+  attr_reader :start_at, :end_at
+
+  def initialize(start_at, end_at)
```
Comment (gabebw) [inline app/models/time_range.rb]: What about using keyword arguments? I think this reads nicely:

```ruby
TimeRange.new(start_at: scheduled_pickup.start_at, end_at: scheduled_pickup.end_at)
```

File: app/helpers/scheduled_pickups_helper.rb [hunk 4]
```diff
@@ -1,14 +1,5 @@
...
-    end_at = scheduled_pickup.end_at
-    time_format = "%l:%M %P"
-
-    t(
-      "scheduled_pickups.format",
-      weekday: Weekday.find(start_at.wday).label,
-      start_at: start_at.strftime(time_format),
-      end_at: end_at.strftime(time_format),
-    )
+    TimeRange.new(scheduled_pickup.start_at, scheduled_pickup.end_at)
```
Comment (kenyonj) [inline app/helpers/scheduled_pickups_helper.rb]: This helper used to return a translation, now it just returns a new object. Are you interpolating that object in the view and is it calling `to_s`?

Comment (gabebw) [inline app/helpers/scheduled_pickups_helper.rb]: Are there tests for this helper method?

Comment (seanpdoyle) [inline app/helpers/scheduled_pickups_helper.rb]: @gabebw this helper (and this newly extracted class) is indirectly tested at the [feature spec level](https://github.com/thoughtbot/freshfoodconnect/blob/1c3b400d3b46ae77aa1db469945fd133243653f2/spec/features/admin_views_pickup_times_spec.rb#L16).

This extraction poses a great opportunity for testing at a more fine-grained level.

Comment (seanpdoyle) [inline app/helpers/scheduled_pickups_helper.rb]: @kenyonj yes, when interpolated, `#to_s` would turn it into the formatted string.

**Comment by seanpdoyle** [comment]
@kenyonj @gabebw I've added some test coverage

**Pull request merged**